TaskVlopper.Base/Logic/IMeetingLogic.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TaskVlopper.Base.Logic
+{
+    public interface IMeetingLogic
+    {
+        IEnumerable<TaskVlopper.Base.Model.Meeting> GetAllMeetingsForCurrentUser(string userId);
+        IEnumerable<TaskVlopper.Base.Model.Meeting> GetAllMeetingsForCurrentUserAndProject(string userId, int projectId);
+        IEnumerable<TaskVlopper.Base.Model.Meeting> GetAllMeetingsForCurrentUserAndProjectAndTask(string userId, int projectId, int taskId);
```

Comment:
Why 3 methods for the same task? Couldn't we use like `GetAllMeetingsForCurrent(string userId = null, int projectId = null...)`?